### ISSUE TITLE
Private instructions on Events not rendered using Markdown

### ISSUE THREAD

**Issue by RichardLitt** [opencollective/opencollective#5461]
**Describe the bug**

The private instructions on events are not rendered using Markdown, as far as I can tell, or they don't match the formatting of the input. 

**To Reproduce**
1. Create an event.
2. Add private instructions with line spaces and more.  
3. See the published page. 

**Expected behavior**

It should use Markdown, I believe? Or a WISYWIG.

**Screenshots**

<img width="1337" alt="Screen Shot 2022-04-21 at 16 33 46" src="https://user-images.githubusercontent.com/910753/164497543-26ebc0fe-4879-4b95-8d43-b62c5bfc2527.png">

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser: Firefox
 - Version: 99.0

<details>
  <summary>Triage Template (core team only)</summary>

  This template is for members of the team to triage for prioritisation. For more guidance see https://www.loom.com/share/369ab467fbc64dec848085d38ff57ca0:

  P1 high frequency, high impact
  P2 low frequency, high impact
  P3 high frequency, low impact
  P4 low frequency, low impact

  Examples of high impact - a problem affects users during an essential process (expenses and payments > onboarding and registration > contributing) with no workaround.

  High frequency - >10% of users affected (measured as a proportion of total potential users for this case).
</details>

**Comment by Betree** [comment]
From what I can see in the code, I don't think this field has ever been treated as something else as simple text (no HTML, no markdown). Did something in the interface made you think otherwise, or is this a feature request?

**Comment by RichardLitt** [comment]
Right! 

Something that made me think it wasn't text was that it supported newlines in the UI, but it doesn't support them in rendering. Perhaps a good halfway point would be to maintain newlines entered in the field in the display?

**Comment by Betree** [comment]
> Right! 
> 
> Something that made me think it wasn't text was that it supported newlines in the UI, but it doesn't support them in rendering. Perhaps a good halfway point would be to maintain newlines entered in the field in the display? 

Absolutely, if it has new lines in the input we must make sure that they're preserved in the frontend **and** in the email (ticket confirmed)

**Comment by RichardLitt** [comment]
Yep. 

Might be worth splitting this into two issues - that newlines aren't preserved I interpret as a bug. Having Markdown enabled is more of a feature request, and may be an issue for email rendering and security, I think.

**Comment by Betree** [comment]
> Yep.
> 
> Might be worth splitting this into two issues - that newlines aren't preserved I interpret as a bug. Having Markdown enabled is more of a feature request, and may be an issue for email rendering and security, I think.

Indeed. For consistency, I'd argue that the feature request should be about having rich text for this field - not markdown - with the WYSIWYG editor that we use in other places. We moved away from markdown a while back because many of our users are non-technical and don't know about this format.

**Comment by RichardLitt** [comment]
Rich text sounds good to me. I've renamed. Would you prefer I re-add the bug label, or make this one about having a WISYWIG?

**Comment by Betree** [comment]
> Rich text sounds good to me. I've renamed. Would you prefer I re-add the bug label, or make this one about having a WISYWIG?

Both work, as you prefer :-)

**Comment by RichardLitt** [comment]
I've received more feedback on this from attendees of the recent @Open-Source-Collective call. The private instructions which included Zoom instructions was not easy for users to navigate via email.
